golden_response/staging/get_stats/census_pep.json

```diff
@@ -123,6 +123,7 @@
       "2019": 328239523
     },
     "place_name": "United States",
+    "is_dc_aggregate": true,
```

Comment:
@shifucun did you add this?

Reply:
hmm, this is odd, you can submit this. I will take a look

Reply:
Ok